MeepMeeptest/.gitignore

```diff
@@ -0,0 +1 @@
+/build
```

Comment:
It looks like extra stuff unrelated to the ticket was checked in, please remove/undo